ichnaea/db.py

```diff
@@ -268,14 +276,16 @@ def create_db(type_=None, uri=None):
         "CREATE DATABASE {} CHARACTER SET = 'utf8'".format(db_to_create)
     )
 
+    alembic_cfg = get_alembic_config()
```

Comment:
For what it is worth, I'm more comfortable with this way of calculating derived settings than the ``MAP_TILES_URL`` setting, seems to be more mockable. But, they both work. 👍